### PR TITLE
Service styling

### PR DESCRIPTION
**Pull request by amosNash**
• Add styling for comparison table

### CODE REVIEW

File: client/style.js [hunk 1]
```diff
@@ -7,6 +7,80 @@ const Title = styled.h1`
...
+
+const PhotoTable = styled.table`
+  border-collapse: collapse;
+`;
+
+const TableData = styled.td`
+  text-align: center;
+`;
+
+const TrEven = styled.tr`
```
Comment (bshresthap) [inline client/style.js:22]: This is for future implementation?

File: client/style.js [hunk 2]
```diff
@@ -7,6 +7,80 @@ const Title = styled.h1`
   border: 5px solid black;
 `;
 
+const CompareTable = styled.table`
```
Comment (bshresthap) [inline client/style.js:10]: Great work with separating styles!

**Review by bshresthap — COMMENTED** [review]
Amos,
Lots of styling in work! Looks good.

File: client/style.js (hunk 1 above)
Comment (amosNash) [reply client/style.js:22]: I didn't end up using this so I will remove it!

File: client/style.js (hunk 2 above)
Comment (amosNash) [reply client/style.js:10]: Thanks! :)

**Pull request merged**